src/module/config.mjs

```diff
@@ -643,8 +643,39 @@ preLocalize("negotation.motivations", {key: "label"});
  * Configuration information for heros
  */
 DRAW_STEEL.hero = {
-  xp_track: [0, 16, 32, 48, 64, 80, 96, 112, 128, 144]
+  /**
+   * XP progression for heroes
+   * @type {number[]}
+   */
+  xp_track: [0, 16, 32, 48, 64, 80, 96, 112, 128, 144],
```

Comment:
I have a general dislike for arrays of "generic" data such as this but I think in this case it's likely the only, or best, way of recording it.